core/domain/takeout_service.py

```diff
@@ -18,15 +18,48 @@
 from __future__ import absolute_import   # pylint: disable=import-only-modules
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
+import inspect
 import re
 
 from core.platform import models
 
 (
-    collection_models, email_models, exploration_models, feedback_models,
-    suggestion_models, user_models) = models.Registry.import_models([
-        models.NAMES.collection, models.NAMES.email, models.NAMES.exploration,
-        models.NAMES.feedback, models.NAMES.suggestion, models.NAMES.user])
+    base_models, collection_models, email_models, exploration_models,
+    feedback_models, suggestion_models,
+    user_models) = models.Registry.import_models(
+        [models.NAMES.base_model, models.NAMES.collection, models.NAMES.email,
+         models.NAMES.exploration, models.NAMES.feedback,
+         models.NAMES.suggestion, models.NAMES.user])
+
+
+def get_models_should_be_exported():
+    """Returns set of strings representing models to export.
```

Comment:
The docstring change from the previous review does not seem to be done yet...

In general, please make sure the code is fixed before replying "Done" to a comment, otherwise it becomes very hard to keep track of previous comments. Thanks!

Reply:
Done